src/components/Shared/FormShell.js

```diff
@@ -279,6 +280,19 @@ export default function FormShell({
               title: platformLabels.ClientBalance
             })
           }
+          onClientSalesTransaction={() =>
```

Comment:
when the RM-75 is merged you need to change this one to be similar to how we are doing the functions in FormShell

Reply:
okay